dashboard/app/models/pd/workshop_material_order.rb

```diff
@@ -50,7 +50,7 @@ class WorkshopMaterialOrder < ActiveRecord::Base
     has_one :workshop, class_name: 'Pd::Workshop', through: :enrollment, foreign_key: :pd_workshop_id
 
     validates :enrollment, presence: true, uniqueness: true
-    validates :user, presence: true, uniqueness: true
+    validates :user, presence: true, uniqueness: true, if: -> {user_id_changed?}
```

Comment:
Yeah, you're right. `user_id_changed?` is false on a new model. However this works: `new_record? || user_id_changed?`. Updating...